Cargo.toml

```diff
@@ -1,18 +1,18 @@
 [package]
 name = "dioxus-helmet"
-version = "0.2.4"
-authors = ["Saicu"]
+version = "0.3.0"
+authors = ["Saicu", "painedpineapple"]
 edition = "2021"
 description = "Small Dioxus component which allows you to place elements in the head of your document"
 license = "MIT"
-repository = "https://github.com/saicu/dioxus-helmet"
+repository = "https://github.com/ranger-finance/dioxus-helmet"
```

Comment:
Wat?

```suggestion
repository = "https://github.com/dioxus-community/dioxus-helmet"
```

Reply:
Yeah, sorry, I wasn't originally planning on creating a PR back into this source repo and forgot to change that back.